superset-embedded-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@superset-ui/embedded-sdk",
-  "version": "0.1.0-alpha.7",
+  "version": "0.1.0-alpha.8",
```

Comment:
I think it's best not to bump this in this PR, but rather when the package is published. WDYT @zhaoyongjie ?

Reply:
@doornot Could you revert this change and `package-lock.json` so that I can merge it even if CI reports that issue?

Reply:
Ok, thanks

Reply:
Ok, thanks @zhaoyongjie